validation/validate.py

```diff
@@ -19,17 +26,29 @@ def validate(collection_path):
             try:
                 in_data = yaml.safe_load(stream)
             except yaml.YAMLError as exc:
-                print(exc)
-
-        # validate it
-        # try:
-        #     CollectionSchema().load(in_data)
-        # except ValidationError as err:
-        #     pprint(err.messages)
-
-        print(f"validating {collection}")
-        parsed = CollectionSchema.parse_obj(in_data)
-        print(parsed)
+                yaml_error_list.append((str(exc.problem) + str(exc.problem_mark)))
+                continue
+
+        # process the Collection
+        try:
+            CollectionSchema.parse_obj(in_data)
+        except ValidationError as err:
+            schema_error_list.append(str(err) + ' in ' + collection)
+
+    if len(yaml_error_list) == 0 and len(schema_error_list) == 0:
+        print("All collections have been validated successfully")
+        return
+
+    if len(yaml_error_list) > 0:
+        print("Yaml Error List")
+        for item in yaml_error_list:
+            print(item)
```

Comment:
It would be nice to have information when there are no yaml_errors. What are your thoughts on:
```
if len(yaml_error_list) > 0:
    print('Yaml Error List')
    print(*yaml_error_list, sep = '\n')
else:    
    print('No Yaml errors')
```

Reply:
Does this apply for `No Schema errors` too?

Reply:
Yeah, I think it could be beneficial for that too. What are your thoughts on it?

Reply:
Sure!